inductiva/structures/scenarios/plate_linear_elastic/mesh_utils.py

```diff
@@ -0,0 +1,276 @@
+"""Utils to create the mesh."""
+
+import time
+from typing import List, Optional
+
+import dolfinx
+from dolfinx.io import gmshio
```

Comment:
This cannot depend on `dolfinx`, since that's the hard dependency that our API solves in this case. In other words, the code that uses `dolfinx` should all be on the API side.